formats/email_alert_signup/publisher/links.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "related"
+  ],
+  "properties": {
```

Comment:
I thought that the idea was that we set to `additionalProperties` to `false` and specify each possible key to avoid typos?